Sources/OpenAI/OpenAI.swift

```diff
@@ -22,14 +22,18 @@ final public class OpenAI: OpenAIProtocol {
         
         /// API host. Set this property if you use some kind of proxy or your own server. Default is api.openai.com
         public let host: String
+
+        /// Optional base path if you set up OpenAI API proxy on a custom path on your own host. Default is ""
+        public let basePath: String?
```

Comment:
What do you think of having a default value in initializer and avoid optional here?